### DISCUSSION TITLE
爆块是概率问题吗？

### DISCUSSION THREAD

**Discussion by love-chia** [Chia-Network/chia-blockchain#2990] [Q&A]
看到群里发的，有的的人仅仅绘了3个图就获得了2个币，有遇到这样情况的吗，

**Comment by anderzheng** [comment]
100块图，跑了一个多星期，一个币也没有。。。

**Reply by HaitaoMiao** [reply]
太难了, 都不够电费的

**Reply by pangchia** [reply]
电费还其次，楼上装修，总断电，nainaide

**Comment by ragnarokna** [comment]
根據文檔描述,是概率問題.每一個圖相當於一張彩票.每十分鐘開32還是64張彩票,從所有圖裡面抽選.中了的得塊,中不著的謝謝參與.
所以必須要在整個網絡上佔有更大的空間百分比才有更大機會得到更多幣.

**Reply by gitscy** [reply]
先要通过初筛, 每张图只有 1 / 512 的概率通过初筛, 通过了才参与抽奖. 也就是说 如果有512张图, 那每次的抽奖活动你都有一张图参与其中.

**Comment by cyr616** [comment]
是的，如果不想碰运气，还是加矿池吧。

**Reply by cca313** [reply]
有什么矿池推荐吗

**Reply by cyr616** [reply]
hpool.com
uupool.com

**Reply by cca313** [reply]
这俩矿池都偷吃啊
